big_tests/tests/jingle_SUITE.erl

```diff
@@ -78,6 +69,46 @@ init_per_suite(Config) ->
             {skip, build_was_not_configured_with_jingle_sip}
     end.
 
+start_nksip_in_mim_nodes() ->
+    Port = 12345,
+    Host = ct:get_config({hosts, mim, domain}),
+    RPCSpec1 = mim(),
+    RPCSpec2 = mim2(),
+    Timeout = timer:seconds(60),
+
+    Pid1 = proc_lib:spawn_link(
+             fun() ->
+                     {ok, _} = rpc(RPCSpec1#{timeout => Timeout}, gen_mod, start_module,
+                                   [
+                                    Host, mod_jingle_sip,
+                                    [{proxy_host, "localhost"},
+                                     {proxy_port, Port},
+                                     {username_to_phone,[{<<"2000006168">>, <<"+919177074440">>}]}]
+                                   ])
+             end),
+    Pid2 = proc_lib:spawn_link(
```

Comment:
Could we have a helper function here and call it with args like (RPCSpec, ExtraOpts)?

```
start_nskip(mim(), []),
start_nksip(mim2(), [{listen_port, 12346}]),
```